### PR TITLE
✨feat(ui)#10274: UI - allow to configure OM logo

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
### Describe your changes :

I worked on issue #10274 and updated the related doc.

**Test cases**
- if `customLogoUrlPath` is present then render the logo from it otherwise render the default logo
- if `customMonogramUrlPath` is present then render the monogram from it otherwise render the default monogram.

> Images used in these screenshots is only for testing purpose.

Login Page
image

Navbar
image

#
### Type of change :

- [x] New feature

#
### Frontend Preview (Screenshots) :

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-docs/content/how-to-guides/custom-logo/how-to-add-custom-logo.md [hunk 1]
```diff
@@ -13,35 +15,26 @@ To change the Logo for the application, we need to update logo at two locations.
...
 <Image src="/images/how-to-guides/custom-logo/nav-Bar-Logo.png" alt="navBar-image"/>
 
-# How to add a custom logo for the application
-
 ### Step 1: Get the image size as per the following formats.
 
-- Login Page (1 px x 2px)
-- Navigation Bar (1 px x 2px)
+- Login Page "Max(150px x 60px)"
+- Navigation Bar "Max(25px x 30px)"
```
Comment (chirag-madlani) [inline openmetadata-docs/content/how-to-guides/custom-logo/how-to-add-custom-logo.md:21]:
```suggestion
- Monogram aspect ratio should be 1:1 and Recommended size should be 30 x 30 px
```

File: openmetadata-docs/content/how-to-guides/custom-logo/how-to-add-custom-logo.md [hunk 2]
```diff
@@ -13,35 +15,26 @@ To change the Logo for the application, we need to update logo at two locations.
 
 <Image src="/images/how-to-guides/custom-logo/nav-Bar-Logo.png" alt="navBar-image"/>
 
-# How to add a custom logo for the application
-
 ### Step 1: Get the image size as per the following formats.
 
-- Login Page (1 px x 2px)
-- Navigation Bar (1 px x 2px)
+- Login Page "Max(150px x 60px)"
```
Comment (chirag-madlani) [inline openmetadata-docs/content/how-to-guides/custom-logo/how-to-add-custom-logo.md:20]:
```suggestion
- Logo aspect ratio should be 3:2 and Recommended size should be 150 x 60 px
```

**Review by chirag-madlani — COMMENTED** [review]
Please update our dimension as per mentioned in the comment

**Comment by Sachin-chaurasiya** [comment]
> Please update our dimension as per mentioned in the comment

@chirag-madlani, I have updated the doc as per the comment, please review.

**Comment by chirag-madlani** [comment]
@Sachin-chaurasiya currently If I have any logo or monogram with mentioned ratio will it work?
Can you verify that? I feel it will break somewhere

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya currently If I have any logo or monogram with mentioned ratio will it work? Can you verify that? I feel it will break somewhere

Sure, I will do that.

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya currently If I have any logo or monogram with mentioned ratio will it work? Can you verify that? I feel it will break somewhere

@chirag-madlani , I have tried with recommended ratio and size, here are the screenshots.

image


image

**Comment by chirag-madlani** [comment]
@Sachin-chaurasiya I just noticed aspect ratio for logo should be 5:2
also try to add bigger image with above ratio and verify like: 500 x 200

**Comment by Sachin-chaurasiya** [comment]
> @Sachin-chaurasiya I just noticed aspect ratio for logo should be 5:2 also try to add bigger image with above ratio and verify like: 500 x 200

@chirag-madlani , Here are the screenshots with bigger image sizes.


image

image

File: docker/local-metadata/docker-compose-postgres.yml [hunk 3]
```diff
@@ -109,6 +109,13 @@ services:
       ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASSWORD:-""}
       # Heap OPTS Configurations
       OPENMETADATA_HEAP_OPTS: ${OPENMETADATA_HEAP_OPTS:--Xmx1G -Xms1G}
+      # Application Config
+      OM_LOGO_LOCATION_TYPE: ${OM_LOGO_LOCATION_TYPE:-openmetadata}
+      OM_LOGO_LOGIN_LOCATION_URL_PATH: ${OM_LOGO_LOGIN_LOCATION_URL_PATH:-""}
+      OM_LOGO_NAVBAR_LOCATION_URL_PATH: ${OM_LOGO_NAVBAR_LOCATION_URL_PATH:-""}
```
Comment (chirag-madlani) [inline docker/local-metadata/docker-compose-postgres.yml:115]: We need to update his as well

File: docker/local-metadata/docker-compose.yml [hunk 4]
```diff
@@ -108,6 +108,13 @@ services:
       ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASSWORD:-""}
       # Heap OPTS Configurations
       OPENMETADATA_HEAP_OPTS: ${OPENMETADATA_HEAP_OPTS:--Xmx1G -Xms1G}
+      # Application Config
+      OM_LOGO_LOCATION_TYPE: ${OM_LOGO_LOCATION_TYPE:-openmetadata}
+      OM_LOGO_LOGIN_LOCATION_URL_PATH: ${OM_LOGO_LOGIN_LOCATION_URL_PATH:-""}
+      OM_LOGO_NAVBAR_LOCATION_URL_PATH: ${OM_LOGO_NAVBAR_LOCATION_URL_PATH:-""}
```
Comment (chirag-madlani) [inline docker/local-metadata/docker-compose.yml:114]: same

File: docker/metadata/docker-compose-postgres.yml [hunk 5]
```diff
@@ -101,6 +101,13 @@ services:
       ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASSWORD:-""}
       # Heap OPTS Configurations
       OPENMETADATA_HEAP_OPTS: ${OPENMETADATA_HEAP_OPTS:--Xmx1G -Xms1G}
+      # Application Config
+      OM_LOGO_LOCATION_TYPE: ${OM_LOGO_LOCATION_TYPE:-openmetadata}
+      OM_LOGO_LOGIN_LOCATION_URL_PATH: ${OM_LOGO_LOGIN_LOCATION_URL_PATH:-""}
+      OM_LOGO_NAVBAR_LOCATION_URL_PATH: ${OM_LOGO_NAVBAR_LOCATION_URL_PATH:-""}
```
Comment (chirag-madlani) [inline docker/metadata/docker-compose-postgres.yml:107]: same

File: docker/metadata/docker-compose.yml [hunk 6]
```diff
@@ -99,6 +99,13 @@ services:
       ELASTICSEARCH_PASSWORD: ${ELASTICSEARCH_PASSWORD:-""}
       # Heap OPTS Configurations
       OPENMETADATA_HEAP_OPTS: ${OPENMETADATA_HEAP_OPTS:--Xmx1G -Xms1G}
+      # Application Config
+      OM_LOGO_LOCATION_TYPE: ${OM_LOGO_LOCATION_TYPE:-openmetadata}
+      OM_LOGO_LOGIN_LOCATION_URL_PATH: ${OM_LOGO_LOGIN_LOCATION_URL_PATH:-""}
+      OM_LOGO_NAVBAR_LOCATION_URL_PATH: ${OM_LOGO_NAVBAR_LOCATION_URL_PATH:-""}
```
Comment (chirag-madlani) [inline docker/metadata/docker-compose.yml:105]: same

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**